lib/msf/ui/tip.rb

```diff
@@ -0,0 +1,38 @@
+# -*- coding: binary -*-
+# frozen_string_literal: true
+
+module Msf
+  module Ui
+    ###
+    #
+    # Module that contains some most excellent tips.
+    #
+    ###
+    module Tip
+      def self.highlight(string)
+        "%grn#{string}%clr"
+      end
+
+      ALL = [
+        "Enable verbose logging with #{highlight('set verbose true')}",
+        "View useful productivity tips with the #{highlight('tip')} highlight, or view them all with #{highlight('tip -l')}",
+        "When in a module, use #{highlight('back')} to unload the module",
+        "Tired of setting RHOST for modules? Try globally setting it with #{highlight('setg RHOST x.x.x.x')}",
+        "Enable http logging with #{highlight('set HttpTrace true')}",
+        "Open an interactive ruby terminal with #{highlight('irb')}",
```

Comment:
Capitalize "ruby," please. A reference to the `pry` command may be helpful, but it may not be what typical module users expect. `irb` is universal.